pkgs/applications/accessibility/espeakup/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, meson, ninja, espeak-ng, fetchFromGitHub, pkg-config, cmake, alsa-lib, systemd }:
+
+stdenv.mkDerivation rec {
+  pname = "espeakup";
+  version = "0.90";
+  src = fetchFromGitHub {
+    owner = "linux-speakup";
+    repo = "espeakup";
+    rev = "569e9f5b0edbf427c5b877912ac4f016efd6a492";
+    sha256 = "sha256:0lmjwafvfxy07zn18v3dzjwwpnid2xffgvy2dzlwkbns8gb60ds2";
+  };
+
+  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
+
+  nativeBuildInputs = [ meson ninja pkg-config cmake ];
```

Comment:
Are you sure `cmake` is needed? Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.